pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+addopts = -rsxX --reuse-db --tb=native
```

Comment:
I had forgotten what `-r` does. It show's some extra test summary for the specified characters. I find listing the skip and xfail issues at the end super helpful.

All the args are expanded here.

I think I want to remove `--reuse-db` and leave that up to devs to do manually. Otherwise, I think this is a good default list.